tests/platform/cnffsdiff/fsdiff.go

```diff
@@ -165,7 +168,25 @@ func (f *FsDiff) RunTest(containerUID string) {
 	}
 
 	f.check.LogInfo("Running \"podman diff\" for container id %s", containerUID)
-	output, err := f.runPodmanDiff(containerUID)
+
+	var output string
+	var err error
+	for i := range [5]int{} {
+		output, err = f.runPodmanDiff(containerUID)
+		if err == nil {
+			break
+		}
```

Comment:
You can combine break with both no error and errors other than 125. There's no need for continue.

Reply:
I'm not quite sure, because we need the continue to restart the loop if we're waiting for error 125 to go away.  

For the first break, that's because there was no error (happy path).  For the second break, that's if the error != nil and doesn't relate to error 125.  

Although now you're making me think about it 😄 I think this should be good.